package.json

```diff
@@ -72,6 +73,9 @@
   "bugs": {
     "url": "https://github.com/doublesymmetry/react-native-track-player/issues"
   },
+  "dependencies": {
+    "shaka-player": "^4.3.2"
```

Comment:
not an expert on deps, but does this meen it gets added a dep for any RNN project as well? shoudl we then perhaps place it in devDeps to mark it as "optional"?

Reply:
BTW I noticed some deprecation warnings from shaka-player in the console. Would be great to fix those imo.

Reply:
@SpadarShut I already fixed the deprecation warning I believe you're referring to [here](https://github.com/doublesymmetry/react-native-track-player/pull/1886/commits/5949f2908ba239d7c1bc24c280c8b0ddfc37c0dc#diff-7380cb609d1f2921eb2360101dbfe66dd422b75ca699917df2d79501dc90d3afR66)

Reply:
@dcvz correct, it'll get installed everywhere. We can move it to an optional dependency, it just means that getting set up with `web` will require a separate installation step. I'll go ahead and do that.

Reply:
@dcvz [fixed here](https://github.com/doublesymmetry/react-native-track-player/pull/1886/commits/ba0fb044b7680b4345908964f2463a537d7f793c)

Reply:
Hmmm... I just realized that removing the package as a dependency causes some typescript errors as we can't access the types at build time. Looking into the best way to solve this.

Reply:
Doh. scratch that. I'll just add it as a `devDependency` as well as an optional `peerDependency`.